### PR TITLE
[wmcb] Cleanup assert usage

### PR DESCRIPTION
**Pull request by vinaykns**
Currently in wmcb, there are few places we are
causing NPE. So this PR will ensure that assert usage
is replaced with require at appropriate places. Also
Nilf is replaces with NoErrorf for more readability.

### CODE REVIEW

**Comment by vinaykns** [comment]
Tested the PR with e2e testing instructions specified in the README.

File: test/e2e/bootstrapper_test.go [hunk 1]
```diff
@@ -76,11 +76,11 @@ func TestBootstrapper(t *testing.T) {
 	// Run it again, to ensure it maintains state if the bootstrapper is already started
 	time.Sleep(5 * time.Second)
 	wmcb, err = bootstrapper.NewWinNodeBootstrapper(installDir, ignitionFilePath, kubeletPath)
-	assert.Nilf(t, err, "Could not create WinNodeBootstrapper: %s", err)
+	assert.NoErrorf(t, err, "Could not create WinNodeBootstrapper: %s", err)
```
Comment (aravindhp) [inline test/e2e/bootstrapper_test.go]: This needs to be `require`. I missed this during the initial review but caught it when rebasing #96.

**Pull request merged**